lldb/include/lldb/Interpreter/CommandInterpreter.h

```diff
@@ -766,6 +771,14 @@ class CommandInterpreter : public Broadcaster,
   CommandUsageMap m_command_usages;
 
   StreamString m_transcript_stream;
+
+  /// Contains a list of handled commands and their details. Each element in
```

Comment:
We should mention the new setting that is required for this to become populated.